src/view/reviewManager.ts

```diff
@@ -764,8 +783,37 @@ export class ReviewManager implements vscode.DecorationProvider {
 				return;
 			}
 
+			let { title } = titleAndDescriptionDefaults;
+
+			const pullRequestNameMethod = await this.getPullRequestNameSetting();
+
+			// User cancelled the name selection process, cancel the create process
+			if (!pullRequestNameMethod) {
+				return;
+			}
+
+			switch(pullRequestNameMethod) {
+				case "branch":
+					if(branchName) {
+						title = branchName;
+					}
+					break;
+				case "custom":
+					const nameResult = await vscode.window.showInputBox({
```

Comment:
I think it would be good to add a `validateInput` option for this input box to prevent the user from entering with an empty title. Would be handled fine below and just halt the whole process, but I think it would be better UX to display a warning in that case, and only treat escape as cancel